crucible/src/Lang/Crucible/CFG/ExtractSubgraph.hs

```diff
@@ -53,6 +53,7 @@ extractSubgraph (CFG{cfgBlockMap = orig, cfgEntryBlockID = _origEntry}) cuts bi
             { cfgBlockMap = bm
             , cfgEntryBlockID = entryID
             , cfgHandle = hn
+            , cfgBreakpoints = undefined
```

Comment:
:-(

Reply:
That's a possibility... I don't remember offhand what it was used for in the first place.